src/main/java/io/github/opencubicchunks/cubicchunks/core/network/WorldEncoder.java

```diff
@@ -51,6 +51,9 @@ static void encodeCubes(PacketBuffer out, Collection<Cube> cubes) {
         cubes.forEach(cube -> {
             out.writeBoolean(cube.isEmpty());
             out.writeBoolean(cube.getStorage() != null);
+            out.writeBoolean(cube.getBiomeArray() != null);
+            out.writeBoolean(false); // reserve
+            out.writeBoolean(false); // reserve
```

Comment:
Each writeBoolean actually writes a whole byte. Which is why I suggested packing multiple flags into one byte, and writing the bytes.